vonk/features/terminology.rst

```diff
@@ -3,7 +3,7 @@
 Terminology
 ===========
 
-.. caution:: Terminology operations in Vonk 3.0.0 only work for STU3. We intend to provide more elaborate terminology support by integrating with a full Terminology Server, including support for R4 in the near future.
+.. caution:: Terminology operations in Vonk 3.0.0 only work for STU3. Starting from Vonk 3.5.0 we provide more elaborate terminology support by integrating with a full Terminology Server, :ref:`Terminology Integration docs<terminologyintegration>`.
```

Comment:
Perhaps add 'see' before the link to the integration docs. The link itself needs to be <feature_terminologyintegration>.